admin_app/src/app/content/edit/page.tsx

```diff
@@ -313,6 +367,7 @@ const ContentBox = ({
                   )
                 ) {
                   event.preventDefault();
+
```

Comment:
remove space?

Reply:
Prettier adds it automatically, is there an issue with the space at the end?